Source/WebCore/platform/graphics/texmap/TextureMapper.h

```diff
@@ -90,12 +90,12 @@ class TextureMapper {
 
     RefPtr<BitmapTexture> applyFilter(RefPtr<BitmapTexture> sourceTexture, const RefPtr<const FilterOperation>&, bool defersLastPass);
 
-    WEBCORE_EXPORT RefPtr<BitmapTexture> acquireTextureFromPool(const IntSize&, const unsigned = 1);
-
 #if USE(GRAPHICS_LAYER_WC)
     WEBCORE_EXPORT void releaseUnusedTexturesNow();
 #endif
 
+    BitmapTexturePool& texturePool() { return *m_texturePool; }
```

Comment:
This doesn't look a good idea to me. The original design hides the implementation detail.
Why do you want to expose BitmapTexturePool?

Reply:
I thought it was clearer than having a method in TextureMapper just to call the pool, but I'm fine with keeping it in the texture mapper so I'll submit a new patch with that part reverted.